### DISCUSSION TITLE
Infinite loading when using SSO link from API

### DISCUSSION THREAD

**Discussion by kobyon** [chatwoot/chatwoot#5441] [Chatwoot APIs]
Hello,
I am running a self hosted chatwoot server and I am trying to generate SSO links for the users.
When I generate the links using curl to interact with the API I get a link SSO link but when opening the link in the browser, the page is spinning an nothing seems to happen

![image](https://user-images.githubusercontent.com/77970971/190414612-2d715c7f-cef9-4f04-aed5-bebc87813741.png)

**Comment by sojan-official** [comment]
@Aresime does the user has access to any account ? also could you check the browser console for errors

**Reply by kobyon** [reply]
Yes the user has access to accounts.  I am can login to the user fine when I am not using SSO link.
The browser console is completely empty.

**Comment by marcipriano** [comment]
I have the same problem. how did you solve it?
